assembly/src/library/mod.rs

```diff
@@ -146,14 +147,14 @@ impl Ord for Module {
 impl Serializable for Module {
     fn write_into<W: ByteWriter>(&self, target: &mut W) {
         self.path.write_into(target);
-        self.ast.write_into(target);
+        self.ast.write_into(target, AstSerdeOptions::new(false));
     }
 }
 
 impl Deserializable for Module {
     fn read_from<R: ByteReader>(source: &mut R) -> Result<Self, DeserializationError> {
         let path = LibraryPath::read_from(source)?;
-        let ast = ModuleAst::read_from(source)?;
+        let ast = ModuleAst::read_from(source, AstSerdeOptions::new(false))?;
         Ok(Self { path, ast })
     }
```

Comment:
I think we should do the same thing here as we did with `ModuleAst` - move this methods into `Module` impl and pass in the `AstSerdeOptions` from the library level.